sdk/keyvault/azure-security-keyvault-administration/CMakeLists.txt

```diff
@@ -39,9 +39,9 @@ if(FETCH_SOURCE_DEPS)
         add_subdirectory(${oneFolder})
     endforeach()
 elseif(NOT AZ_ALL_LIBRARIES)
-  find_package(azure-core-cpp "1.5.0" CONFIG QUIET)
```

Comment:
2.8.0 was just made available 28 minutes ago: https://github.com/microsoft/vcpkg/pull/29427

Reply:
not working 

....
The following packages will be built and installed:
    azure-core-cpp[core,curl,http,winhttp]:x64-windows -> 1.7.2
....

Reply:
CI vcpkg cache needs to update

Reply:
I think, in `eng/pipelines/templates/jobs/cmake-generate.tests.yml`, we need to add a step to run `vcpkg update`, before the `cmake --version` step.

Reply:
this us getting more and more complicated, i'd rather we update to 1.8.0 after the release. This thing does not use any new features so it doesn't really need it . also i suspect we would need to update the baselines for vcpkg that are in varios configs in 2 places the vcpkg baselines , another change. see how this keeps spiraling for no good reason right now .